### PR TITLE
[MRG] increase legal-weight meta page

### PR DESCRIPTION
**Pull request by michelfernandez**
- Added new page for increase legal-weight in DE
- Added some new images
- Optimized title by enable-qes for DE, FR, EN
- Changed graphic aes-for-business DE
- Added new menu point on navigation de: increase legal-weight

### CODE REVIEW

File: site/config.yaml [hunk 1]
```diff
@@ -40,13 +40,17 @@ languages:
           url: "/de/identifikation/"
           weight: 30
           parent: plattform
+        - name: Beweikraft erhöhen
```
Comment (betatim) [inline site/config.yaml:43]:
```suggestion
        - name: Beweiskraft erhöhen
```

File: site/config.yaml [hunk 2]
```diff
@@ -78,12 +82,15 @@ languages:
         - name: Identifikation
           url: /de/identifikation/
           weight: 20
+        - name: Beweikraft erhöhen
```
Comment (betatim) [inline site/config.yaml:85]:
```suggestion
        - name: Beweiskraft erhöhen
```

File: site/config.yaml (hunk 1 above)
Comment (ronioeschger) [inline site/config.yaml:43]:
```suggestion
        - name: Beweiskraft erhöhen
```

File: site/config.yaml (hunk 2 above)
Comment (ronioeschger) [inline site/config.yaml:85]:
```suggestion
        - name: Beweiskraft erhöhen
```

**Comment by fursinger** [comment]
@dkumpf1 will test this PR and give the tested: OK label if the tests are successful.

**Comment by dkumpf1** [comment]
I found those two bugs:

1. It says "AES" in an image on the AES for Business-page
2. Wrong language as you can see in the second screenshot

If there is nothing against those two bugs, then it can be released and I will set the label.
cove_aes_for_biz_1
cove_aes_for_biz_2

**Comment by michelfernandez** [comment]
Thanks guys! I corrected the 2 issues.

**Pull request merged**